relay-server/src/actors/processor.rs

```diff
@@ -563,6 +564,7 @@ impl EnvelopeProcessorService {
         project_cache: Addr<ProjectCache>,
         global_config: Addr<GlobalConfigManager>,
         upstream_relay: Addr<UpstreamRelay>,
+        _aggregator: Addr<Aggregator>,
```

Comment:
```suggestion
        #[cfg(feature = "processing")]
        aggregator: Addr<Aggregator>,
```

Reply:
I followed the convention with _redis there, but if i'll change mine to cfg then i might as well do the same with redis?